src/config/config.mlh.tpl

```diff
@@ -0,0 +1,46 @@
+(* GENERATED FILE - DO NOT EDIT *)
+(* Profile: {PROFILE} *)
+[%%define ledger_depth {LEDGER_DEPTH}]
+[%%define curve_size {CURVE_SIZE}]
+[%%define coinbase "{COINBASE}"]
+[%%define consensus_mechanism "{CONSENSUS_MECHANISM}"]
```

Comment:
This technically won't work with our nonconsensus build, but I think we can use the literal `none` string instead with very few changes, which would make this just work.

Leaving this comment to give me somewhere to link to from the issue I'll be creating.